lametro/models.py

```diff
@@ -487,6 +487,25 @@ class LAMetroEvent(Event, LiveMediaMixin, SourcesMixin):
     class Meta:
         proxy = True
 
+    @classmethod
+    def most_recent_past_meetings(cls):
+        '''
+        Returns meetings in the current month that have occured in the past
+        two weeks.
+        '''
+        current_month = timezone.now().month
+        two_weeks_ago = timezone.now() - timedelta(weeks=2)
+
+        meetings_in_past_two_weeks = cls.objects.filter(
+            Q(start_time__month=current_month), Q(start_time__gte=(two_weeks_ago)), Q())
+
+        # since display_status is a property of LAMetroEvent rather than
+        # a model attribute, we have to make sure returned meetings 
+        # have concluded separately from the above Queryset filter
+        past_meetings = list(filter(lambda m: m.display_status == 'Concluded', meetings_in_past_two_weeks))
```

Comment:
Thoughts on using `has_passed` directly, rather than `display_status`, which uses it to determine whether a meeting has concluded?

Reply:
Good call.